app/views/challenge_signups/_signup_form.html.erb

```diff
@@ -22,7 +22,9 @@
 <% end %>
 
 <%= form_for([@collection, @challenge_signup], :url => (@challenge_signup.new_record? ? collection_signups_path(@collection) : collection_signup_path(@collection))) do |signup_form| %>
-  <p class="notes"><%= ts("Challenge maintainers may have access to the email address associated with your AO3 account for the purpose of communicating with you about the challenge.") %></p>
+  <% if @collection.challenge_type == "GiftExchange" %>
+   <p class="notes notice"><%= ts("Challenge maintainers will have access to the email address associated with your AO3 account for the purpose of communicating with you about the challenge.") %></p>
```

Comment:
Very minor style thing: can you indent the `<p>` line by 4 spaces, the ` <% end %>` line by 2?

Reply:
yup. does that look right now?